python/langsmith/_internal/_operations.py

```diff
@@ -5,6 +5,8 @@
 import uuid
 from typing import Literal, Optional, Union, cast
 
+import zstandard as zstd
```

Comment:
Do we actually want this to be a strict dependency?

Reply:
made optional